sdk/python/tests/compiler/testdata/parallel_join.py

```diff
@@ -0,0 +1,41 @@
+from kfp_tekton.compiler import TektonCompiler
+from kfp import dsl
+
+
+def gcs_download_op(url):
+    return dsl.ContainerOp(
+        name='GCS - Download',
+        image='google/cloud-sdk:216.0.0',
+        command=['sh', '-c'],
+        arguments=['gsutil cat $0 | tee $1', url, '/tmp/results.txt'],
+    )
+
+
+def echo_op(text):
+    return dsl.ContainerOp(
+        name='echo',
+        image='library/bash:4.4.23',
+        command=['sh', '-c'],
+        arguments=['echo "$0"', text]
+    )
+
+
+@dsl.pipeline(
+    name='Parallel and Sequential pipeline',
```

Comment:
`    name='Parallel and Sequential pipeline',`
-->
`    name='Parallel join pipeline',`
or `    name='Parallel pipeline',` ... to keep consistent with the [KFP example](https://github.com/kubeflow/pipelines/blob/e344b74dfd6bf316900ed808a363de30d953b9f2/samples/core/parallel_join/parallel_join.py#L42)